### ISSUE TITLE
Parallel background compilation triggers assertion failure

### ISSUE THREAD

**Issue by RalfJung** [ProofGeneral/PG#119]
Steps to reproduce:
* Enable "Compile before require" and "Compile parallel in background"
* Clone `git-rts@gitlab.mpi-sws.org:FP/iris-coq.git`, checkout out commit `b863b045333`
* Run `make prelude/tactics.vo`
* Open file `prelude/tactics.v`
* Go to the first blank line (after the imports) and hit `C-C C-Ret`

Expected behavior:
* All dependencies are already compiled, so it should step to where the cursor is

Actual behavior:
Emacs prints
```
unexpected error during parallel compilation: (cl-assertion-failed (<= (length result) 1) 2)
Kill all Coq background compilation processes
byte-code: Assertion failed: (<= (length result) 1), 2
```

This is with PG 25355560b

**Comment by RalfJung** [comment]
Wow, that was quick. I can confirm the fix. Thanks a lot :-)

**Comment by hendriktews** [comment]
Ralf,

> unexpected error during parallel compilation: (cl-assertion-failed (<= (length result) 1) 2)

it's the vio line of coqdep that causes this problem. I just
merged the fix for this into master. Could you try with version
8851ccfc7ae8d439b45b56053cf93462f948d85d (I don't have a password
yet for your gitlab)?

Hendrik

**Comment by RalfJung** [comment]
You don't need a password for our GitLab, anonymous cloning should work... I just gave you the wrong URL for that, sorry :/  This one should do it: <https://gitlab.mpi-sws.org/FP/iris-coq.git>

Anyway, when I said "I can confirm the fix" above, I meant to say that I updated PG and the issue is fixed :)  That was with 8851ccfc indeed.
